index.js

```diff
@@ -311,16 +311,14 @@ class ResembleHelper extends Helper {
       options.tolerance = 0;
     }
 
-    const prepareBaseImage = options.prepareBaseImage !== undefined
-      ? options.prepareBaseImage
-      : (this.prepareBaseImage === true)
     const awsC = this.config.aws;
-    if (awsC !== undefined && prepareBaseImage === false) {
-      await this._download(awsC.accessKeyId, awsC.secretAccessKey, awsC.region, awsC.bucketName, baseImage, options);
-    }
-    if (options.prepareBaseImage !== undefined && options.prepareBaseImage) {
+
+    if (this._getPrepareBaseImage(options)) {
```

Comment:
This change also addresses the following bug report:

https://github.com/codeceptjs/codeceptjs-resemblehelper/issues/88